src/pages/AddItem/styles.css

```diff
@@ -1,8 +1,42 @@
+.add-container {
+  border-radius: 25px;
+  height: 90vh;
+  width: 500px;
+}
+
 .add-form {
+  text-align: left;
   align-items: center;
   flex-direction: column;
   justify-content: center;
   width: fit-content;
   display: flex;
-  margin: 0 auto 30px;
+  margin: 30px auto 5px;
+  height: 60vh;
+}
+
+.button-addform {
+  background-color: #f2b807;
+  border: none;
+  color: #000;
+  text-decoration: none;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  font-size: 25px;
+  margin-bottom: 40px;
+  border-radius: 8px;
+  width: 100%;
+  height: 40px;
+}
```

Comment:
Maybe you can add some `padding` to your button, and change your `font-size`.

For example:

```css
.button-addform {
    //.... more styles
    font-size: 16px;
    padding: 7px;
}
```

Reply:
Done, 
thanks for the recommendation